cassandra/cqlengine/columns.py

```diff
@@ -332,7 +332,7 @@ def __init__(self, min_length=None, max_length=None, **kwargs):
         :param int max_length: Sets the maximum length of this string, for validation purposes.
         """
         self.min_length = (
-            1 if not min_length and kwargs.get('required', False)
+            1 if min_length is not None and kwargs.get('required', False)
```

Comment:
I think it should be `1 if min_length is None and kwargs.get('required', False)`